.github/workflows/test.yml

```diff
@@ -52,6 +53,16 @@ jobs:
       run: |
         python bin/sample_build.py
 
+    - name: Get some sample wheels
+      run: |
+        python -m test.test_projects test.test_0_basic.basic_project sample_proj
+        cibuildwheel --output-dir wheelhouse sample_proj
+
+    - uses: actions/upload-artifact@v2
```

Comment:
There shouldn't be any harm in always having this, I'd think.

Reply:
Yeah.. maybe let's leave it in, at least until we can test these universal2/arm64 wheels in CI.